src/WinRPM.jl

```diff
@@ -48,11 +48,11 @@ end
         if VERSION >= v"0.3-"
             res = ccall((:URLDownloadToCacheFileW,:urlmon),stdcall,Cuint,
               (Ptr{Void},Ptr{Uint16},Ptr{Uint16},Clong,Cint,Ptr{Void}),
-              0,utf16(source),dest,sizeof(dest)>>1,0,0)
+              C_NULL,utf16(source),dest,sizeof(dest)>>1,0,C_NULL)
         else
```

Comment:
It doesn't look like this branch should ever be taken, since `REQUIRE` specifies julia 0.3?